database/migrations/2022_05_11_201130_add_full_name_col.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddFullNameCol extends Migration
+{
+	private const USERS = 'users';
+	private const NAME = 'fullname';
+
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::table(self::USERS, function (Blueprint $table) {
+			$table->string(self::NAME, 128)->default('')->after('password')->nullable();
```

Comment:
If the column is nullable - which is fine - then the default should not be the empty string `''`. As we use `null` for "unknown value" or "not set", we should do that here, too.

Reply:
It is the most robust definition for a column. I changed it.